### PR TITLE
Bomb

### PR DESCRIPTION
**Pull request by Alexosk**

### CODE REVIEW

**Review by mangster80 — APPROVED** [review]
Noice!

**Review by mirandamutka — APPROVED** [review]

File: html/script/map2.js [hunk 1]
```diff
@@ -217,4 +218,41 @@ function showPosition (position, gameMap) {
...
+function disarmBomb () {
+  let clue1 = document.forms['clueForm']['clue1'].value
+  let clue2 = document.forms['clueForm']['clue2'].value
+  let clue3 = document.forms['clueForm']['clue3'].value
+  let clue4 = document.forms['clueForm']['clue4'].value
+  let clue5 = document.forms['clueForm']['clue5'].value
+  if (clue1 === answer[0] && clue2 === answer[1] && clue3 === answer[2] && clue4 === answer[3] && clue5 === answer[4] && timer > 0) {
+    alert('Du har vunnit')
+  } else {
+    alert('Du har förlurat')
```
Comment (falt) [inline html/script/map2.js]: Change spelling to "förlorat"

**Review by falt — CHANGES_REQUESTED** [review]

File: html/script/map2.js (hunk 1 above)
Comment (Alexosk) [reply html/script/map2.js]: Fixad

**Review by falt — APPROVED** [review]

**Review by Svahnen — APPROVED** [review]

**Comment by Svahnen** [comment]
https://cdn.discordapp.com/attachments/389719765217705986/405657299143753728/iconer.png

**Pull request merged**